examples/internal/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -160,6 +160,26 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 			}
 		}
 	}
+	responses: {
+		key: "500";
+		value: {
+			description: "Server error";
+			headers: {
+				key: "x-correlation-id"
+				value: {
+					description: "Unique event identifier for server requests\n\nExample: `2438ac3c-37eb-4902-adef-ed16b4431030`\n\nPattern: `^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$`"
```

Comment:
I'm confused by this formatting, is the pattern and example supposed to be embedded in the description like this?

Reply:
No I will remove that from the example... this is something I am doing because the redoc-cli(swagger2openapi) tool drops both of those fields in the conversion.

Reply:
No... I can remove this. This is actually something that I am doing because the redoc-cli(swagger2openapi) conversion drops these fields.